modules/oe_starter_content_event/config/install/field.field.node.oe_event.oe_featured_media.yml

```diff
@@ -0,0 +1,23 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.oe_featured_media
+    - node.type.oe_event
+id: node.oe_event.oe_featured_media
+field_name: oe_featured_media
+entity_type: node
+bundle: oe_event
+label: 'Featured media'
+description: ''
+required: false
+translatable: false
+default_value: {  }
+default_value_callback: ''
+settings:
+  handler: 'default:media'
+  handler_settings:
+    target_bundles:
+      av_portal_photo: av_portal_photo
+      image: image
```

Comment:
Lets ask @brummbar if we need to add also videos, if yes we also need it in news

Reply:
When Francesco requested oe_media, he requested av_portal_photo and media enabled for the featured media field and not videos.

Reply:
was discussed, ok